components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -1247,6 +1250,76 @@ def render_col_plot(request, iid, z, t, x, w=1, conn=None, **kwargs):
 
 @login_required()
 @jsonp
+def api_container_list(request, conn=None, **kwargs):
+    # Get parameters
+    try:
+        page = getIntOrDefault(request, 'page', 1)
+        limit = getIntOrDefault(request, 'limit', settings.PAGE)
+        group_id = getIntOrDefault(request, 'group', -1)
+        experimenter_id = getIntOrDefault(request, 'id', -1)
+    except ValueError:
```

Comment:
How do you know which param is incorrect if you catch them all together without exposing that in error message?